ui/app/helpers/mountable-secret-engines.js

```diff
@@ -1,29 +1,31 @@
 import { helper as buildHelper } from '@ember/component/helper';
 
-export const KMIP = {
-  displayName: 'KMIP',
-  value: 'kmip',
-  type: 'kmip',
-  category: 'generic',
-  requiredFeature: 'KMIP',
-};
-
-export const TRANSFORM = {
-  displayName: 'Transform',
-  value: 'transform',
-  type: 'transform',
-  category: 'generic',
-  requiredFeature: 'Transform Secrets Engine',
-};
-
-export const KEYMGMT = {
-  displayName: 'Key Management',
-  value: 'keymgmt',
-  type: 'keymgmt',
-  glyph: 'key',
-  category: 'cloud',
-  requiredFeature: 'Key Management Secrets Engine',
-};
+const ENTERPRISE_SECRET_ENGINES = [
+  {
+    displayName: 'KMIP',
+    value: 'kmip',
+    type: 'kmip',
+    engineRoute: 'kmip.scopes',
+    category: 'generic',
+    requiredFeature: 'KMIP',
+  },
+  {
+    displayName: 'Transform',
+    value: 'transform',
```

Comment:
if `value` and `type` are the same for each engine...do we need both keys on each of these objects?

Reply:
Hm, this file was only rearranged but I will look into that